docs/building.md

```diff
@@ -24,19 +24,30 @@ git submodule update --recursive
 
 LLVM builds take a long time, so we cached the build directory from Linux, Windows and MacOS for our CI, that can also be used on developer's machines.
 
-Run the following script at the root of your checkout:
+The CMake flag `VERONA_DOWNLOAD_LLVM` is used to control whether a cached build 
+is used or not. 
+
 ```
-bash ./utils/llvm/setup-llvm-builddir.sh verona-llvm-x86_64-${OS}-${BuildType}-${LLVMCommit}.tar.gz
+cmake -DVERONA_DOWNLOAD_LLVM=On ..
 ```
+Will automatically pull in a pre-compiled install directory for LLVM, and use
+that for building Verona.
+This is the default.
 
-With `${OS}` = { `linux`, `windows`, `macos` }, `BuildType` = { `release`, `debug` } and `${LLVMCommit}` being the same as the llvm-project submodule (currently @ `3c123acf57c`).
-
-On Windwows, run it on PowerShell, not `cmd`, to get access to unix-like tools. You can use LLVM release builds on Linux and MacOS debug builds of Verona, but on Windows, you must use the same build type, to avoid CRT incompatibility issues.
-
-If you want to compile LLVM directly, the options we used in the cache are available [here][]. For now, you'll have to build it before Verona in its own build directory. We're working to make that build straight from Verona's build directory, but it's not yet ready.
-
-[here]: https://github.com/microsoft/verona/blob/master/devops/llvm.yml
+If this flag is unset, 
+```
+cmake -DVERONA_DOWNLOAD_LLVM=On ..
```

Comment:
This part is ambiguous about whether this comment unsets the flag or whether it will result in compiling LLVM locally

Reply:
Fixed